hello-oboe/app/src/main/cpp/OboeSinePlayer.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+
+#ifndef HELLO_OBOE_OBOESINEPLAYER_H
+#define HELLO_OBOE_OBOESINEPLAYER_H
+
+
+#include <math.h>
+#include <algorithm>
+
+#include <oboe/Oboe.h>
+
+class OboeSinePlayer: public oboe::AudioStreamCallback {
+public:
+
+    OboeSinePlayer() {
+        oboe::AudioStreamBuilder builder;
+        // The builder set methods can be chained for convenience.
+        builder.setSharingMode(oboe::SharingMode::Exclusive)
+                ->setPerformanceMode(oboe::PerformanceMode::LowLatency);
```

Comment:
Please could we avoid the `->` syntax here. It's more readable to just use `builder.set*`